src/screens/HomeScreen/HowYouFeel.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import {
+  Text,
+  View,
+  AsyncStorage,
+  StyleSheet,
+  FlatList,
+  Image,
+  Dimensions,
+  TouchableOpacity,
+} from 'react-native';
+
+export default class HowYouFeel extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      emoticons: [
+        require('../../assets/images/Emoji/neutral.png'),
```

Comment:
1. Please don't use the required on. Please Import all images. 
2. Start using the conventions as they configured in `babel.config.js`. If the file under the `src` then use `@/folderName/fileName`. This is called module resolver. 
```
import neutralImage from '@/assets/images/Emoji/neutral.png'
```